Sources/PostgreSQL/ConnectionInfo.swift

```diff
@@ -0,0 +1,29 @@
+#if os(Linux)
+    import CPostgreSQLLinux
+#else
+    import CPostgreSQLMac
+#endif
+
+public enum ConnInfo {
```

Comment:
I'm not in favour of abbreviations even though it is very clear, but I understand it would become quite long.